README.md

```diff
@@ -1,4 +1,4 @@
-# [<img src="https://github.com/mineiros-io/brand/blob/38f6e5c20a176c62dc673f707a3987c9ec3691cd/terramate-vertical-logo-black-font.svg" height="120"/>](https://mineiros.io/?ref=terramate)[<img align="right" src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" height="23" alt="Mineiros"/>](https://mineiros.io/?ref=terramate)
+# [<img src="https://github.com/mineiros-io/brand/blob/38f6e5c20a176c62dc673f707a3987c9ec3691cd/terramate-vertical-logo-black-font.svg#gh-light-mode-only" height="120"/><img src="https://raw.githubusercontent.com/mineiros-io/brand/master/terramate-vertical-logo-white-font.svg#gh-dark-mode-only" height="120"/>](https://mineiros.io/?ref=terramate)[<img align="right" src="https://raw.githubusercontent.com/mineiros-io/brand/3bffd30e8bdbbde32c143e2650b2faa55f1df3ea/mineiros-primary-logo.svg" height="23" alt="Mineiros"/>](https://mineiros.io/?ref=terramate)
```

Comment:
hmm for the other images we are using permalinks but then for the dark mode one we are referencing master directly, which I assume is not what we want :thinking:

Reply:
I got a permalink for the branch I was playing with: https://raw.githubusercontent.com/mineiros-io/brand/3a3bf4c1a401128715765c4725e168aa3e04c893/terramate-vertical-logo-white-font.svg#gh-dark-mode-only